int16[] data			# [pixX, pixY, theta]
